tests/acceptance/editor.feature

```diff
@@ -22,3 +22,12 @@ Background:
     And I publish the post
     Then I see the video in the editor
     And I see the video in the post published
+
```

Comment:
I think I need to create a `campaign.feature` file and add all campaign related stuff in this file. 🤔